src/module_handle.cc

```diff
@@ -0,0 +1,279 @@
+#include "module_handle.h"
+#include "context_handle.h"
+#include "external_copy.h"
+#include "reference_handle.h"
+#include "isolate/class_handle.h"
+#include "isolate/run_with_timeout.h"
+#include "isolate/three_phase_task.h"
+
+#include <map>
+#include <mutex>
+
+using namespace v8;
```

Comment:
You're using the v8 namespace here but there's like 50 "v8::" specifiers in this file. I'd say drop the `v8::` litter from the rest of the file :)